fastparquet/converted_types.py

```diff
@@ -115,6 +115,7 @@ def convert(data, se):
     elif ctype == parquet_thrift.ConvertedType.INTERVAL:
         # for those that understand, output is month, day, ms
         # maybe should convert to timedelta
+        # TODO: seems like a np.view should do this much faster
```

Comment:
My understanding is that interval is used for time ranges, like "from 12:00 to 1:00pm".  Not sure what the state of Pandas support is for this.  cc @TomAugspurger